diff --git a/pages/intro.mdx b/pages/intro.mdx
index d220384..fda14d2 100644
--- a/pages/intro.mdx
+++ b/pages/intro.mdx
@@ -2,6 +2,4 @@ import Counter from '~/components/Counter.jsx'
 
 # Introduction
 
-This is an example page written in Markdown!!
-
 <Counter/>
diff --git a/src/App.vue b/src/App.vue
index 1b411bc..f9387b7 100644
--- a/src/App.vue
+++ b/src/App.vue
@@ -1,7 +1,7 @@
 <template>
-  <!-- <img alt="Vue logo" src="./assets/logo.png"/>
+  <img alt="Vue logo" src="./assets/logo.png"/>
   <Counter/>
-  <HelloWorld msg="Hello Vue 3 + Vite"/> -->
+  <HelloWorld msg="Hello Vue 3 + Vite"/>
   <intro/>
 </template>
 
